qgis-app/styles/models.py

```diff
@@ -0,0 +1,94 @@
+from django.db import models
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+
+STYLES_STORAGE_PATH = getattr(settings,
+                                 'PLUGINS_STORAGE_PATH', 'styles/%Y')
+
+
+class StyleType(models.Model):
+    """
+    Style Type model
+    """
+
+    # symbol, name and desc
+    # symbol e.g. "line"
+    # name e.g. "Line"
+    symbol = models.CharField(_('Symbol'),
+                            help_text=_('Must be unique'),
```

Comment:
For help text, the text should be descriptive and informative, telling the user what the field is for. Its ok to include the uniqueness requirement but the other descriptive aspect should also be covered. Applies to all other help texts in this PR